<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Testcase_table</class>
 <widget class="QMainWindow" name="Testcase_table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用例表</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QPushButton" name="Btn_cancel_case">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Btn_set_case">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>配置</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QTreeWidget" name="TreeWidget_case">
      <property name="setItemWidget" stdset="0">
       <string>hello</string>
      </property>
      <column>
       <property name="text">
        <string>用例标题</string>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="background">
        <color>
         <red>255</red>
         <green>170</green>
         <blue>127</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="200">
          <red>85</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
      </column>
      <item>
       <property name="text">
        <string>通话测试</string>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <item>
        <property name="text">
         <string>主叫主挂</string>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
       </item>
       <item>
        <property name="text">
         <string>主叫被挂</string>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
       </item>
       <item>
        <property name="text">
         <string>主叫拒接</string>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
       </item>
       <item>
        <property name="text">
         <string>主叫未接</string>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>卡测试</string>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <item>
        <property name="text">
         <string>SIM卡检测</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SD卡检测</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>WIFI测试</string>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
       </property>
       <item>
        <property name="text">
         <string>WIFI开关</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
        <property name="flags">
         <set>ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WIFI连接</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>蓝牙测试</string>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="checkState">
        <enum>Unchecked</enum>
       </property>
       <item>
        <property name="text">
         <string>蓝牙开关</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
       </item>
       <item>
        <property name="text">
         <string>蓝牙连接</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
       </item>
      </item>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="Btn_save_case">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
